README.md

```diff
@@ -50,6 +50,7 @@ order = public.place_order(
     quantity=1,
     side='BUY', # or 'SELL'
     order_type='MARKET', # or 'LIMIT' or 'STOP'
+    limit_price=None # pass if using 'LIMIT' order_type
```

Comment:
I'd make a note that this can be a float/decimal value. Like "pass float if using..."